### DISCUSSION TITLE
AVG detecting dfdata.bella.network as infected (Phishing)

### DISCUSSION THREAD

**Discussion by abelljms** [Thomas2500/uDomainFlag#8] [Q&A]
so avg flag you as a virus, I've emailed you too.
![avg240730b prob](https://github.com/user-attachments/assets/898d9143-5f98-473f-a44b-1e3ce9c1cb1e)
![avg240731a grr](https://github.com/user-attachments/assets/ebad6d31-08b8-4e09-bd6c-1428da246b9c)
![avg240731b grr](https://github.com/user-attachments/assets/e371a3d9-cd89-437a-9584-b974265fdb1e)
****

**Comment by abelljms** [comment]
AVG yesterday were adamant they are right

**Comment by Thomas2500** [comment]
Hi @abelljms!
Thanks for the report and detailed info!

The target address https://dfdata.bella.network/flags/companysync always reponds with the content "`{"enabled":false}`" which indicates the extension to not use any company regulations - this functionality is described here: https://github.com/Thomas2500/uDomainFlag/blob/4e8b6d03c5c473d149da660464f09de00c2bc59f/script/domainflag.js#L567-L580

Any different response would break the functionality of uDomainFlag.

I already contacted AVG of this false-positive, unfortunately I haven't received any response from them. I have therefore sent another request to AVG so that the error can be checked as quickly as possible.

As of the access and auditing logs of my server, nothing was compromised and no unauthorized access happened. The only difference is a migration to a new server with a new IP 2 weeks ago. I assume that AVG has an error here, that the IP address now used was misused by the previous owner months ago for spam/phishing and AVG is using outdated data for detection.

I will keep you updated as soon as I receive an answer from AVG here.

**Comment by abelljms** [comment]
Many thanks for this.
  

  
It's only flagging via Chrome, so I have removed the adding temporarily.
  

  
I will also press AVG to reverify your domain etc.
  

  
JM Swallow   
  

  

  

  
  
  
  
>
